testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -28,8 +28,12 @@ Feature: PXE boot a Retail terminal
     And I manually install the "vsftpd" formula on the server
     And I manually install the "saltboot" formula on the server
     And I manually install the "pxe" formula on the server
+    And I manually install the "image-sync" formula on the server
     And I synchronize all Salt dynamic modules on "proxy"
 
+  Scenario: Restart spacewalk services to apply saltboot reactor configuration
+    When I restart the spacewalk service
```

Comment:
Is this something the customer must apply, and is it documented?

If not, we should avoid such "cheating" because we know about the internals.

If this is to work around a bug, please document it as such:
```
# WORKAROUND: bsc#xxxxx
```

Reply:
Documentation says that after installing retail pattern, reboot is required:
https://github.com/uyuni-project/uyuni-docs/blob/master/modules/retail/pages/retail-install-uyuni.adoc

This is equivalent to reboot.

Reply:
I don't see similar steps in https://documentation.suse.com/suma/4.3/en/suse-manager/retail and I'm confused.